tiledb/tests/test_cloud.py

```diff
@@ -15,7 +15,10 @@
 
 
 @pytest.mark.skipif(
-    not os.getenv("CI") and tiledb_token is None,
+    os.getenv("CI") == None
```

Comment:
I'm pretty sure this will be set in the nightly setup?

Reply:
Yes, but the others will not be set.

Reply:
The env var `CI` is set to `true` by default on GitHub Actions

https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/store-information-in-variables#default-environment-variables